### ISSUE TITLE
Kobo Forma: initial scan fails

### ISSUE THREAD

**Issue by hrdl-github** [koreader/lj-wpaclient#14]
https://github.com/koreader/lj-wpaclient/pull/13 / https://github.com/koreader/lj-wpaclient/commit/0d5c8ee336b699dbb35850a7336d24e89e1aac24 don't work on my Kobo Forma as intended (https://github.com/koreader/koreader/issues/12848). Instead of `SCANNING` my `wpa_state` seems to be `INACTIVE` when scanning. The initial connection usually takes around 20 seconds, as  described in https://github.com/koreader/lj-wpaclient/issues/12.

**Comment by NiLuJe** [comment]
On which FW version?

With an AP svaed in Nickel?

With an AP saved in Koreader?

In range of said AP?

**Comment by NiLuJe** [comment]
I assume things get better if you actually get Nickel to save the AP?

**Comment by NiLuJe** [comment]
Nope, we usually just added some logging around the scan heuristics in
lj-wpaclient. There might even be commented out code for that in there ;).

**Comment by NiLuJe** [comment]
Huh. The whole point of https://github.com/koreader/lj-wpaclient/pull/13 was *exactly* to handle that initial `INACTIVE` state when there aren't any APs configured, so I'm curious where the heuristic is falling on its ass here...

(i.e., uncomment the two debug `print` calls, and perhaps add another one that does something along the lines of `print(status and status.wpa_state or nil)` next to the last one).

Either something's wonky in the current heuristic, or we just need to be more aggressive with that check (as discussed in the PR).

**Comment by NiLuJe** [comment]
> Something like this:
> 
> ```
> SCAN reply:     OK
> done=nil, found_result=nil, incoming=false
> getStatus reply line:   wpa_state=INACTIVE
> ```

Well, that should return early, that was the whole point of #13 o_O.

> how about

Too platform-specific ;).
